api/rpc/stat/influx/stat_test.go

```diff
@@ -29,11 +30,14 @@ func TestQuery(t *testing.T) {
 }
 
 func statsInit() {
+	//Need to sleep for CI swarm to launch stacks
+	time.Sleep(1000 * time.Millisecond)
```

Comment:
Don't do this. If you can't connect, fail fast with a panic. To get CI build passing is a responsibility outside of our server. I'm adding a script to make sure upstream services are healthy before running `make test`.